js/TodoList.js

```diff
@@ -0,0 +1,84 @@
+function TodoList({onToggle, onRemove, onMod}) {
+    const $todoList = document.querySelector("#todo-list");
+
+    $todoList.addEventListener("click", event => {
+        let target = event.target;
+
+        if(target.classList.contains("toggle")){
+            onToggleTodoItem(target);
+        }else if(target.classList.contains("destroy")){
+            onRemoveTodoItem(target);
+        }
+    });
+
+    $todoList.addEventListener("dblclick", event => {
+        onChangeToEditing(event.target);
+    });
+
+    $todoList.addEventListener("keydown", event => {
+        if(event.key === "Escape"){
+            onChangeToView(event.target);
+        }else if(event.key === "Enter"){
+            onModTodoItem(event.target);
+        }
+    })
```

Comment:
event를 사용하지 않는다면 매개변수를 `({key, target})` 과 같이 넣어서 간단하게 표현할 수 있어요 :frog:
```suggestion
    $todoList.addEventListener("keydown", ({key, target}) => {
        if(key === "Escape"){
            onChangeToView(target);
        }else if(key === "Enter"){
            onModTodoItem(target);
        }
    })
```

Reply:
늦게 PR 올려서 리뷰가 없을 줄 알았는데 리뷰해주셔서 감사드립니다.
리뷰해주신 거 잘 참고하고 기회가 있으면 다음 미션도 진행하도록 하겠습니다~

Reply:
저도 늦게 리뷰를 달아서 혹시 확인 안하셨을까봐 걱정했었습니다. 조금이라도 도움이 되셨으면 좋겠네요 :frog: